### PR TITLE
[run-int-test][OSD-12227] update int-test bootstrap script for local & staging

### PR DESCRIPTION
**Pull request by rajdeepc2792**
Integration test bootstrap script updates for local and cluster flows.

- Sanity Check added for checking:-
   (1) Current Directory == Root Directory
   (2) AWS credential variables are set
 
 - backplane-cluster-admin access for staging flow.

### CODE REVIEW

File: hack/scripts/integration-test-bootstrap.sh [hunk 1]
```diff
@@ -274,8 +296,13 @@ function profileProw {
 }
 
 function profileStage {
+    clusterUserName=$($OC whoami)
+    $OC --as backplane-cluster-admin adm policy add-cluster-role-to-user cluster-admin $clusterUserName
```
Comment (macgregor) [inline hack/scripts/integration-test-bootstrap.sh:300]: looks good. just add a comment here about why youre doing this.

**Review by macgregor — CHANGES_REQUESTED** [review]
minor request

File: hack/scripts/integration-test-bootstrap.sh (hunk 1 above)
Comment (rajdeepc2792) [reply hack/scripts/integration-test-bootstrap.sh:303]: Added the comment.

**Pull request merged**